external/js/kyber/lib/sign/schnorr/schnorr.js

```diff
@@ -0,0 +1,128 @@
+const group = require("./../../group");
+// XXX to be changed to an interface style
```

Comment:
@gnarula is also doing the work of putting an generic interface, like Kyber. Once ready i'll move this code towards using it.